OpenEdXMobile/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,626 @@
+  <string name="edit_user_profile_current_location">Localização atual: {current_location}</string>
+  <!--Button label that will open gallery or camera for choosing new profile photo-->
+  <string name="edit_user_profile_change_photo">Alterar</string>
+  <!--Content Description for change photo button-->
+  <string name="edit_user_profile_change_photo_description">Altere a foto do perfil</string>
+  <!--Button that will open a camera for taking a new profile photo-->
+  <string name="edit_user_profile_take_photo">Tirar foto</string>
+  <!--Button that will open photo gallery for choosing a new profile photo-->
+  <string name="edit_user_profile_choose_photo">Escolher foto</string>
+  <!--Button that will open a camera for taking a new profile photo-->
+  <string name="edit_user_profile_remove_photo">Remover foto</string>
+  <!--Title when cropping newly selected profile photo-->
+  <string name="edit_user_profile_photo_crop_title">Cortar e redimensionar</string>
+  <!--Button that will exit the crop tool without saving the image-->
+  <string name="edit_user_profile_photo_crop_cancel">Cancelar</string>
+  <!--Button that will confirm the crop region and continue to upload the cropped photo-->
+  <string name="edit_user_profile_photo_crop_save">Escolher</string>
+  <!--Tab title for section of profile that displays a user's bio-->
+  <string name="profile_tab_bio">Sobre</string>
+  <!--Tab title for section of profile that lists a user's accomplishments-->
+  <string name="profile_tab_accomplishment">Conquistas</string>
+  <!--Button that will initiate sharing one of a user's accomplishments-->
+  <string name="share_accomplishment_button">Compartilhe suas conquistas</string>
+  <!--Text that pre-populates your post/email/SMS when sharing certificate-->
+  <string name="share_accomplishment_message">Acabei de ganhar uma medalha em {platform_name}! Confira: {badge_url}</string>
+  <!--Question on the app rating dialog-->
+  <string name="rating_dialog_message">Qual é a sua avaliação para o aplicativo?</string>
+  <!--Dialog title for positive app review-->
+  <string name="rate_app_dialog_title">Avalie o aplicativo</string>
+  <!--Dialog message for positive app review-->
+  <string name="rate_app_dialog_message">Compartilhe sua opinião sobre o app {platform_name} escrevendo uma rápida avaliação.</string>
+  <!--Dialog title for negative app review-->
+  <string name="feedback_dialog_title">Enviar comentário?</string>
+  <!--Dialog message for negative app review-->
+  <string name="feedback_dialog_message">Ajude-nos a melhorar</string>
+  <!--accessibility label for rating bar-->
+  <string name="rating_bar">Controle de avaliação</string>
+  <!--accessibility label for rating selection on rating bar-->
+  <string name="rating_bar_selection">{rating} de {num_of_stars}</string>
+  <!--Title of the Whats New screen e.g. Version 2.8.1 New Features-->
+  <string name="whats_new_title">Versão {app_version}, Novos recursos</string>
```

Comment:
it should be `version_number` instated of `app_version`, otherwise app will crash when user install the app from the scratch, and mobile language is already `Portuguese (Brazil)`